inc/class-antivirus.php

```diff
@@ -146,6 +146,13 @@ public static function activation() {
 		if ( self::_cron_enabled( self::_get_options() ) ) {
 			self::_add_scheduled_hook();
 		}
+
+		// Add admin notice and disable the feature, if Safe Browsing is enabled without custom API key.
+		$safe_browsing_key = self::_get_option( 'safe_browsing_key' );
+		if ( self::_get_option( 'safe_browsing' ) && empty( $safe_browsing_key ) ) {
+			self::_update_option( 'safe_browsing', 0 );
+			set_transient( 'antivirus-activation-notice', true, 2592000 );
+		}
```

Comment:
In my manual tests the admin notice was never shown. When (re)activating AntiVirus, all options are cleared in ll. 138-143. Therefore the condition in l. 152 can never be true, right?

This check be run independent of activation, shouldn't it? I. e. after the update or in the `check_safe_browsing()` method itself? Then we even cover cases such as updating incl. the correction + warning and importing a DB backup afterwards - check is repeated and results in showing the warning again.

Reply:
Works for me in the intended way:
* save settings with SB enabled and no API key in v1.4.3 (raises a warning)
* update plugin to 1.5.0
* deactivate and activate the plugin (done automatically when updating the regular way)
* warning is shown _once_ that SB has been disabled

<details><summary>screenshot (animated)</summary>

![antivirus-104](https://user-images.githubusercontent.com/12963621/119983949-21b47500-bfc1-11eb-9c44-ab2c5f7c724d.gif)

</details>

Reply:
> warning is shown once that SB has been disabled

Could this be displayed until it is actively dismissed? Otherwise it could easily be missed ...